### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by unidrelyn**
## Summary 
Styles of the 'tech' file are changed, the css grid is used

## Screenshot 
image

## Trello Card
https://trello.com/c/mvIEuRlu/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: Maquetacion/scss/tech.scss [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
+    padding: 5px;
     width: 220px;
 
-    &:hover {
-      box-shadow: 10px 10px $white, 10px 10px 0 1px $green;
+    &:first-child {
+      border-right: 2px solid $green;
+      grid-area: bigitem;
+      height: 100%;
+      padding: 160px 5px;
```
Comment (jmtov) [inline Maquetacion/scss/tech.scss]: Why did you set this huge padding? If it's just for centering the content of the tech card, you can use another positioning technique based in grid/flex. :)

File: Maquetacion/index.html [hunk 2]
```diff
@@ -57,8 +57,8 @@ <h2 class="text title-card">Polymer</h2>
       </div>
     </div>
     <h1 class="title">Back-End Tools</h1>
-    <div class="front">
-      <div class="tech">
+    <div class="container">
+      <div class="tech item1">
```
Comment (jmtov) [inline Maquetacion/index.html]: You can remove the `item1` class name if you're not using it anywhere. 👍

**Review by jmtov — CHANGES_REQUESTED** [review]
Nice approach with the pseudo-selectors! Please check out my comments 😺

**Review by FrankIglesias — APPROVED** [review]

**Comment by FrankIglesias** [comment]
Correct the @jmtov changes and go!

File: Maquetacion/scss/tech.scss (hunk 1 above)
Comment (unidrelyn) [reply Maquetacion/scss/tech.scss]: the padding is corrected by a display: grid

File: Maquetacion/index.html (hunk 2 above)
Comment (unidrelyn) [reply Maquetacion/index.html]: 👍

**Review by jmtov — APPROVED** [review]

**Pull request merged**